apps/styleguide/src/tables/RenderValue.vue

```diff
@@ -1,23 +1,37 @@
 <template>
-  <span v-if="row[col.name] && Array.isArray(row[col.name])">
-    <span v-for="(val, idx) in row[col.name]" :key="idx">
-      {{ col.refLabel ? applyJsTemplate(col.refLabel, val) : val }}
+  <span v-if="row[col.id] && Array.isArray(row[col.id])">
+    <span v-for="(val, idx) in row[col.id]" :key="idx">
+      <a v-if="col.columnType == 'HYPERLINK_ARRAY'" :href="val" target="_blank">
```

Comment:
can you assume val is valid href?

Reply:
val has been validated on input, so it should be valid. 
I (we?) didn't know about that display, will change it :)